internal/definitions/handler_tokenpool.go

```diff
@@ -52,7 +52,7 @@ func (dh *definitionHandler) handleTokenPoolDefinition(ctx context.Context, stat
 
 	pool.Namespace = dh.namespace.Name
 	if err := pool.Validate(ctx); err != nil {
-		return HandlerResult{Action: ActionReject, CustomCorrelator: correlator}, i18n.NewError(ctx, coremsgs.MsgDefRejectedValidateFail, "token pool", pool.ID, err)
+		return HandlerResult{Action: ActionReject, CustomCorrelator: correlator}, i18n.WrapError(ctx, err, coremsgs.MsgDefRejectedValidateFail, "token pool", pool.ID)
```

Comment:
I don't think this should change (unless it changes everywhere)... `MsgDefRejectedValidateFail` has 3 substitution params in the string (for the type, the ID, and the validation error)

Reply:
(I'm also OK with changing it everywhere if that seems to make the most sense... just need to adjust the error string and everywhere else it's used)